app/feature/feature-home/src/main/kotlin/com/hedvig/android/feature/home/home/data/GetHomeDataUseCase.kt

```diff
@@ -48,9 +42,7 @@ internal class GetHomeDataUseCaseImpl(
         .fetchPolicy(if (forceNetworkFetch) FetchPolicy.NetworkOnly else FetchPolicy.CacheFirst)
         .safeFlow(::ErrorMessage),
       getMemberRemindersUseCase.invoke(),
-      flow { emit(getTravelCertificateSpecificationsUseCase.invoke().getOrNull()) },
-      flow { emit(featureManager.isFeatureEnabled(Feature.MOVING_FLOW)) },
-    ) { homeQueryDataResult, memberReminders, travelCertificateData, isMovingFlowEnabled ->
+    ) { homeQueryDataResult, memberReminders ->
```

Comment:
Yes, my bad, I removed travelCertificateData bc it's not used at home anymore, and removed unused  featureManager as well. I'll merge develop in feature now

Reply:
Yeah overall it's encouraged to sometimes if it makes sense to do some slight cleanup even if it's not necessarily 100% related with what you are working on.
This is not a problem at all in this PR, I only mention it since I know that we fixed this on develop too anyway, so it gives the merge conflict, that's all 😊